.github/workflows/unit-tests.yml

```diff
@@ -17,7 +17,21 @@ jobs:
         with:
           cache: yarn
           node-version: '16'
+      - run: yarn global add depcheck
       - run: yarn install --frozen-lockfile
+      - name: Generate .depcheckrc.json
+        run: node scripts/generateDepcheckrc.js
+      - name: Run depcheck
+        working-directory: ./packages/jaeger-ui
+        run: |
+          depcheck
+          DEPCHECK_STATUS=$?
+          if [ $DEPCHECK_STATUS -ne 0 ]; then
+            echo "Unused dependencies found by depcheck."
+            exit 1  # Exit with a failure status
+          fi
+      - name: Remove .depcheckrc.js
+        run: rm packages/jaeger-ui/.depcheckrc.json
```

Comment:
This is not a good way to do this, i.e. have the logic only in GH workflow. As this is a linter step we should be able to run it locally via `yarn depcheck`, so can we move this all into package.json/scripts?

Reply:
moved to package.json to work as a yarn script